data/transport missions.txt

```diff
@@ -1975,3 +1975,40 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Loan Broker Reunion"
+	name "The Broker"
+	description "Transport the broker's enforcers to <destination> by <day>."
+	blocked "You'll need more room to house the hired thugs. Return when you have <capacity>."
```

Comment:
Since this is only a spaceport mission (as it doesn't have a `job` or `landing` tag), it doesn't make sense to have a `blocked` line.